src/libs/actions/Policy.ts

```diff
@@ -3565,6 +3565,64 @@ function enablePolicyTags(policyID: string, enabled: boolean) {
 }
 
 function enablePolicyTaxes(policyID: string, enabled: boolean) {
+    const taxRatesData: OnyxData = {
+        optimisticData: [
+            {
+                onyxMethod: Onyx.METHOD.MERGE,
+                key: `${ONYXKEYS.COLLECTION.POLICY}${policyID}`,
+                value: {
+                    taxRates: CONST.DEFAULT_TAX.RATES,
+                },
+            },
+            {
+                onyxMethod: Onyx.METHOD.MERGE,
+                key: `${ONYXKEYS.COLLECTION.POLICY}${policyID}`,
+                value: {
+                    taxRates: {
+                        taxes: {
+                            ...Object.keys(CONST.DEFAULT_TAX.RATES.taxes).reduce(
+                                (prevTaxesData, taxKey) => ({
+                                    ...prevTaxesData,
+                                    [taxKey]: {pendingAction: CONST.RED_BRICK_ROAD_PENDING_ACTION.ADD},
+                                }),
+                                {},
+                            ),
+                        },
+                    },
+                },
+            },
```

Comment:
No. Because:
1. When we send ```taxFields``` param to BE, we will send ```CONST.DEFAULT_TAX.RATES```. If we combine like you mentioned, the redundant field ```pendingAction``` is sent.
2. When defining a ```CONST.DEFAULT_TAX.RATES```, if we combine the ```pendingAction``` data in here, we need to hard code the ```pendingAction: 'add'```

Reply:
I'm not sure that I get that. We send this to the API:

```
parameters.taxFields = JSON.stringify(CONST.DEFAULT_TAX.RATES);
```
What I'm suggesting wouldn't change `CONST.DEFAULT_TAX.RATES`.

Reply:
Left a suggestion